packages/fxa-shared/subscriptions/stripe.ts

```diff
@@ -231,3 +259,13 @@ export const getSubscriptionUpdateEligibility: (
 
   return SubscriptionUpdateEligibility.INVALID;
 };
+
+export const getMinimumAmount = (currency: string): number => {
+  if (STRIPE_MINIMUM_CHARGE_AMOUNTS[currency]) {
+    return STRIPE_MINIMUM_CHARGE_AMOUNTS[currency];
+  }
+
+  throw new Error(
+    `Currency "${currency}" does not have a minimum amount available.`
+  );
```

Comment:
This is ok for now since I don't think anything is catching this error.  But generally speaking, determining the type of error by checking the error message text is not ideal.